src/configuration/routes.md

```diff
@@ -117,11 +117,11 @@ If your project involves only a single apex domain with one app or multiple apps
 
 ## Route identifiers
 
-All routes defined on an environment will be available in the application in the `PLATFORM_ROUTES` environment variable.  That value is a base64-encoded JSON string, which can be extracted in the language of your choice to look up the effective routes that were generated.
+All routes defined for an environment are available to the application in its `PLATFORM_ROUTES` environment variable, which contains a base64-encoded JSON object. This object can be parsed by the language of your choice to give your application access to the generated routes.
 
-"Effective route" here means that all placeholders will be replaced with the appropriate domain name, and depending on your configuration additional route entries may be generated as well (such as the automatic HTTP->HTTPS redirect).  That means the domains will vary from one environment to another.  To make it easier to locate specific routes in a standardized fashion you may also specify an `id` on each route that is stable across all environments.  You may also mark one, and only one route as `primary`.  The primary route may be special cased in the interface but has no impact on the runtime environment.
+When routes are generated, all placeholders will be replaced with appropriate domains names, and if your configuration specifies them, additional route entries may be generated (e.g. automatic HTTP to HTTPS redirects). To make it easier to locate routes in a standardized fashion, you may specify an `id` key on each route which remains stable across environments. You may also specify a single route as `primary`, which will cause it to be highlighted in the web interface but will have no impact on the runtime environment.
```

Comment:
HTTP->HTTPS redirects are only generated automatically if they're NOT specified in the file.  The change in the first sentence is thus backwards.